crapssim/strategy.py

```diff
@@ -461,6 +461,26 @@ def place68_dontcome2odds(player, table, unit=5, strat_info=None):
         if not player.has_bet("LayOdds") and not dc.prepoint:
             player.bet(LayOdds(mult * unit, dc))
 
+def dontpass_2come_dontcome(player, table, unit=5, strat_info=None):
+    # start with dontpass bet
+    # once the point is established, place come bet twice
+    # finally place a dont come bet
+    # whenever there is less than 2 come bet replenish it.
+    dontpass(player, table, unit)
+
+    if table.point == "On":
+        total_come_bets = player.has_bet("Come")

```

Comment:
player.has_bet returns a boolean and won't get the total amount of come bets on the table. You are probably looking for something like:
```
total_come_bets = len([bet for bet in player.bets_on_table if bet.name == "Come"])
```